db/version_edit.h

```diff
@@ -311,6 +311,13 @@ class VersionEdit {
     remaining_entries_ = remaining_entries;
   }
 
+  bool GetNextFileNumber(uint64_t* result) const {
```

Comment:
Let not add this method, but calling `HasNextFile()` and `GetNextFile()` combination on rocksdb_cloud side instead.